app/services/hyrax/workflow/workflow_factory.rb

```diff
@@ -2,7 +2,7 @@ module Hyrax
   module Workflow
     class WorkflowFactory
       class_attribute :workflow_strategy
-      self.workflow_strategy = DefaultWorkflowStrategy
+      self.workflow_strategy = Hyrax::Workflow::WorkflowByAdminSetStrategy
```

Comment:
Two questions:

1. I wonder if this means we should also dump the `DefaultWorkflowStrategy` since it's no longer being used?
2. Update line 9 below?